events/wc_harpy_tribal_events.txt

```diff
@@ -0,0 +1,372 @@
+######################Cipher Event - Harpy Events#############################
+namespace = WCHO
+
+#Mating event post sieging
+character_event = {
+	id = WCHO.1
+	title = EVTTITLE_WCHO.1
+	desc = EVTDESC_WCHO.1
+	picture = GFX_evt_harpy_siege
+	border = GFX_event_normal_frame_diplomacy
+	
+	is_triggered_only = yes
+	hide_from = yes
+	
+	trigger = {
+		ROOT = { 
+			trait = creature_harpy
+			culture = harpy
+		}
+		FROM = { 
+			owner = {
+				any_courtier = {
+					at_location = FROM
+					is_adult = yes
+					is_possible_harpy_victim_trigger = yes

```

Comment:
Since `is_possible_harpy_victim_trigger` isn't needed. Read my comment above.
```suggestion
					ROOT = { can_has_sexual_relation_with_prev = yes }
```